### ISSUE TITLE
Need to add WCAG tests and fix any missing items. 

### ISSUE THREAD

**Issue by ogdenstudios** [ogdenstudios/recipeswithoutblogs#5]
Accessibility is top priority, and ensuing we're at least WCAG AA compliant is top of the list, here. We need to figure out the best way to integrate WCAG AA testing with all deploys to ensure this standard, and make any changes to the site to comply.
